### PR TITLE
feat(main-titles):add titles to the main categorys

### LINKED ISSUE #21: Display category titles bases on what category was searched
**Is your feature request related to a problem? Please describe.**
There is no title telling the user where they are when they press a category button.

**Describe the solution you'd like**
For that category's title to be displayed on screen when a user presses a category button

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
1. Added titles to the main categories
2. Colored them to match their look on the nav buttons
3. Added each their own images from the nav buttons

## Purpose
Make the main page look better and let the user know which page they are on

Closes #21

### CODE REVIEW

File: modern-news-app/src/components/Main.tsx [hunk 1]
```diff
@@ -36,21 +52,43 @@ const Main: React.FC<MainProps> = ({ newsData }) => {
...
+            return <TitleStyled style={{ color: "#6F81D8"}}>Business<img src={business} alt="stock chart"/></TitleStyled>
+        } else if(newsData.search === "health"){
+            return <TitleStyled style={{ color: "#00C0C8"}}>Health<img src={health} alt="heart"/></TitleStyled>
+        } else if(newsData.search === "world"){
+            return <TitleStyled style={{ color: "#FF9A86"}}>World<img src={world} alt="world"/></TitleStyled>
+        } else {
+            return <TitleStyled>You searched for "{newsData.search}"</TitleStyled>
+        }
+    }
+
```
Comment (JayDampitan) [inline modern-news-app/src/components/Main.tsx:70]: Very nice!

Comment (krferrell) [inline modern-news-app/src/components/Main.tsx:70]: Looks good! Could use a switch statement here to clean the code up a bit.

**Review by krferrell — APPROVED** [review]
Nice work.

**Review by JayDampitan — APPROVED** [review]
Good job Luis!

File: modern-news-app/src/components/Main.tsx (hunk 1 above)
Comment (Lap343) [reply modern-news-app/src/components/Main.tsx]: I thought the same exact thing when I pushed it. New code is up and it does look better

**Pull request merged**